addons/binding/org.openhab.binding.chromecast.test/pom.xml

```diff
@@ -0,0 +1,66 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.openhab.binding</groupId>
+	<artifactId>org.openhab.binding.chromecast.test</artifactId>
+
+	<packaging>eclipse-test-plugin</packaging>
+	<name>chromecast Binding Tests</name>
+
+	<parent>
+		<groupId>org.openhab.binding</groupId>
+		<artifactId>pom</artifactId>
+		<version>2.0.0-SNAPSHOT</version>
+	</parent>
+
+
+
+	<properties>
+		<bundle.symbolicName>org.openhab.binding.chromecast.test</bundle.symbolicName>
+		<bundle.namespace>org.openhab.binding.chromecast.test</bundle.namespace>
+	</properties>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-surefire-plugin</artifactId>
+				<version>${tycho-version}</version>
+				<configuration>
+					<dependencies>
+						<dependency>
```

Comment:
Please note that adding dependencies like this might not work as reliably as expected. Instead, you might want to declare them as extra-requirements in the target platform plugin and only set the start level in the surefire plugin. 

Also, mind that the declarative services bundle needs to be started before all the others in order to properly work.

```xml
      <plugin>
        <groupId>${tycho-groupid}</groupId>
        <artifactId>target-platform-configuration</artifactId>
        <configuration>
          <environments combine.self="override"></environments>
          <dependency-resolution>
            <extraRequirements>
              <requirement>
                <type>eclipse-plugin</type>
                <id>org.eclipse.equinox.ds</id>
                <versionRange>0.0.0</versionRange>
              </requirement>
              <requirement>
                <type>eclipse-plugin</type>
                <id>org.eclipse.equinox.event</id>
                <versionRange>0.0.0</versionRange>
              </requirement>
              <requirement>
                <type>eclipse-plugin</type>
                <id>org.eclipse.smarthome.config.xml</id>
                <versionRange>0.0.0</versionRange>
              </requirement>
              <requirement>
                <type>eclipse-plugin</type>
                <id>org.eclipse.smarthome.core.thing.xml</id>
                <versionRange>0.0.0</versionRange>
              </requirement>
              <requirement>
                <type>eclipse-plugin</type>
                <id>org.eclipse.smarthome.core.binding.xml</id>
                <versionRange>0.0.0</versionRange>
              </requirement>
            </extraRequirements>
          </dependency-resolution>
        </configuration>
      </plugin>
      <plugin>
        <groupId>${tycho-groupid}</groupId>
        <artifactId>tycho-surefire-plugin</artifactId>
        <configuration>
          <bundleStartLevel>
            <bundle>
              <id>org.eclipse.equinox.ds</id>
              <level>1</level>
              <autoStart>true</autoStart>
            </bundle>
            <bundle>
              <id>org.eclipse.equinox.event</id>
              <level>2</level>
              <autoStart>true</autoStart>
            </bundle>
            <bundle>
              <id>org.eclipse.smarthome.config.xml</id>
              <level>4</level>
              <autoStart>true</autoStart>
            </bundle>
            <bundle>
              <id>org.eclipse.smarthome.core.thing.xml</id>
              <level>4</level>
              <autoStart>true</autoStart>
            </bundle>
            <bundle>
              <id>org.eclipse.smarthome.core.binding.xml</id>
              <level>4</level>
              <autoStart>true</autoStart>
            </bundle>
          </bundleStartLevel>
        </configuration>
      </plugin>
```